src/common/options/global.yaml.in

```diff
@@ -4673,6 +4673,70 @@ options:
   default: 5
   see_also:
   - bluestore_cache_autotune
+- name: bluestore_cache_autotune_kv_intervals
+  type: str
+  level: dev
+  desc: A 10 element, space separated list of age intervals for kv cache
+  fmt_desc: |
+    A 10 element, space separated list of cache age intervals grouped by 
+    priority such that PRI1=[0,n), PRI2=[n,n+1), PRI3=[n+1,n+2) ... 
+    PRI10=[n+8,n+9).  Values are specified in seconds. A 0 in the 2nd term
+    will prevent any items from being associated with that bin. For example,
```

Comment:
Values are not specified in seconds.
Values are specified in bluestore_cache_autotune_interval units.

Reply:
I think it should be in seconds.
We could of course provided dedicated configuration that will clock bins, but I think it would be overkill.

It should not dependent on cache_autotune_interval, since time base for ageing of bins is important, and cache_autotune_interval is just a update frequency.

I think balance() should be invoked as is, but `rotate_bins()` should be invoked 1/s.
I guess it would mean some math on cache_autotune_interval, or is it acceptable that `rotate_bins()` are not invoked precisely?

Reply:
I think the intervals can be independent.  When we balance we just look at whatever the most recently updated state of the caches were.  It doesn't really matter if it was 1 second ago or 4 seconds ago.  The bigger advantage by separating them is that we can allow for much higher resolution on the age bins.  Instead of an autotune_interval minimum bin size, it can now be as small as you want to make it (with 1 second being a nice easy default imho).